src/frontend/src/btc/utils/btc-transactions.utils.ts

```diff
@@ -16,11 +16,51 @@ export const mapBtcTransaction = ({
 	btcAddress: BtcAddress;
 	latestBitcoinBlockHeight: number;
 }): BtcTransactionUi => {
-	const isTypeSend = inputs.some(({ prev_out }) => prev_out.addr === btcAddress);
-	const output: BitcoinOutput | undefined = out.find(({ addr }) =>
-		isTypeSend ? addr !== btcAddress : addr === btcAddress
+	const { totalInputValue, isTypeSend } = inputs.reduce<{
+		totalInputValue: number;
+		isTypeSend: boolean;
+	}>(
+		(acc, { prev_out: { value, addr } }) => ({
+			totalInputValue: acc.totalInputValue + value,
+			isTypeSend: !acc.isTypeSend ? addr === btcAddress : acc.isTypeSend
```

Comment:
i know it is just a detail eheh, but maybe invert the condition for readability (for me)?

Reply:
Sure 🙂